test/TestProjects/CadlFirstTest/Generated/cadl.json

```diff
@@ -0,0 +1,183 @@
+{
+  "Name": "demo.helloworld",
+  "Description": "",
+  "ApiVersions": ["0.1.0"],
+  "Clients": [
+    {
+      "Name": "HelloWorld",
+      "Operations": [
+        {
+          "Name": "topAction",
+          "Description": "",
+          "Parameters": [
+            {
+              "$id": "1",
+              "Name": "Endpoint",
+              "NameInRequest": "Endpoint",
+              "Description": "",
+              "Type": {"Name": "Uri", "Kind": "Uri"},
+              "Location": "Uri",
+              "IsRequired": true,
+              "IsEndpoint": true,
+              "Kind": "Client"
+            },
+            {
+              "Name": "action",
+              "NameInRequest": "action",
+              "Description": "",
+              "Type": {"Name": "string", "Kind": "String"},
+              "Location": "Path",
+              "IsRequired": true,
+              "IsApiVersion": false,
+              "IsResourceParameter": false,
+              "IsContentType": false,
+              "IsEndpoint": false,
+              "Kind": "Method",
+              "SkipUrlEncoding": true,
+              "Explode": false
+            },
+            {
+              "$id": "2",
+              "Name": "apiVersion",
+              "NameInRequest": "apiVersion",
+              "Description": "",
+              "Type": {"Name": "string", "Kind": "String"},
+              "Location": "Query",
+              "IsRequired": true,
+              "IsApiVersion": true,
+              "Kind": "Client"
+            }
+          ],
+          "Responses": [
+            {
+              "StatusCodes": [
+                200
+              ],
+              "BodyType": {"Name": "Thing", "Kind": "Object"},
```

Comment:
Is this how models are defined?  Will it be possible to define them once in a common place?

Reply:
Yes. There will be a `Models` section